### PR TITLE
Fix/telephony ccs batch2 fixes

### PR DESCRIPTION
**Pull request by frenautvh**
| Question         | Answer
| ---------------- | ---
| Branch?          | fix/telephony-ccs-fixes
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | internal MANAGER-6651
| License          | BSD 3-Clause

## Description

batch 2 of CCS refactoring/rework : various UX improvements & fixes

### CODE REVIEW

File: packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -102,6 +102,7 @@
   "telephony_number_feature_ovh_pabx_step_rule_ivr": "Lancer un menu interactif",
   "telephony_number_feature_ovh_pabx_step_rule_ivr_explain": "Permet de proposer à l'appelant un menu avec lequel il pourra interagir grâce aux touches de son téléphone.",
   "telephony_number_feature_ovh_pabx_step_rule_ivr_menu_to_launch": "Menu à lancer",
+  "telephony_number_feature_ovh_pabx_step_rule_ivr_menu_already_added": "Menu déjà ajoutés",
```
Comment (antleblanc) [inline packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/translations/Messages_fr_FR.json:105]: Not needed to add it into `Messages_**_**.json`.

**question:** Shouldn't be this instead?
```suggestion
  "telephony_number_feature_ovh_pabx_step_rule_ivr_menu_already_added": "Menu déjà ajouté",
```

File: packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/sound/list/list.html [hunk 2]
```diff
@@ -14,13 +14,13 @@
                     data-ng-disabled="$ctrl.loading.init || !$ctrl.ovhPabx.sounds.length"
                     data-ng-click="$ctrl.orderDesc = !$ctrl.orderDesc"
                 >
-                    <span data-translate="common_lastname"></span>
+                    <span data-translate="common_lastname"> </span>
```
Comment (antleblanc) [inline packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/sound/list/list.html:17]: Extra space needed?

File: packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/rule/edit/edit.html [hunk 3]
```diff
@@ -594,18 +593,17 @@
...
-                                </span>
                             </div>
                         </div>
                     </button>
                     <!-- END OF ADD NEW MENU -->
                 </div>
 
                 <div class="section-group">
+                    <div
+                        class="group-title ng-scope"
```
Comment (antleblanc) [inline packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/rule/edit/edit.html:604]:
```suggestion
                        class="group-title"
```

File: packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/ovhPabx/tts/tts.html [hunk 4]
```diff
@@ -44,7 +44,7 @@ <h1 data-translate="telephony_alias_ovh_pabx_tts_title"></h1>
         class="row"
         data-ng-if="!$ctrl.loading.init && $ctrl.number.getFeatureFamily() === 'ovhPabx' && $ctrl.number.feature.isCcs()"
     >
-        <div class="col-xs-12 col-md-6 widget-presentation">
+        <div class="col-xs-12 col-md-12 widget-presentation">
```
Comment (antleblanc) [inline packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/ovhPabx/tts/tts.html:47]: **suggestion:** Using a `.row > col-**-12` means that we can refer to a single `div`.

File: packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/ovhPabx/tts/translations/Messages_fr_CA.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
-  "telephony_alias_ovh_pabx_tts_breadcrumb": "Gestion des TTS",
-  "telephony_alias_ovh_pabx_tts_title": "Gestion des TTS (Text To Speech)",
+  "telephony_alias_ovh_pabx_tts_breadcrumb": "Gérer des TTS",
+  "telephony_alias_ovh_pabx_tts_title": "Gérer des TTS (Text To Speech)",
```
Comment (antleblanc) [inline packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/ovhPabx/tts/translations/Messages_fr_CA.json:3]: Especially for this translation, shouldn't be done into the `Messages_fr_FR.json` instead?

File: packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/ovhPabx/tts/translations/Messages_fr_FR.json [hunk 6]
```diff
@@ -1,5 +1,5 @@
 {
-  "telephony_alias_ovh_pabx_tts_breadcrumb": "Gestion des TTS",
+  "telephony_alias_ovh_pabx_tts_breadcrumb": "Gérer des TTS",
   "telephony_alias_ovh_pabx_tts_title": "Gestion des TTS (Text To Speech)",
```
Comment (antleblanc) [inline packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/ovhPabx/tts/translations/Messages_fr_FR.json:3]:
```suggestion
  "telephony_alias_ovh_pabx_tts_title": "Gérer des TTS (Text To Speech)",
```

**Review by antleblanc — CHANGES_REQUESTED** [review]
Well done @frenautvh!

I've left a few comments that are not blocking but I think there is a small requested changes regarding translation.

Please let me know. Thank you!

File: packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/agents/ovhPabx/ovh-pabx.html [hunk 7]
```diff
@@ -61,47 +73,35 @@ <h1 data-translate="telephony_alias_configuration_agents_title"></h1>
...
-                    data-ng-click="AgentsOvhPabxCtrl.addAgentForm.numbers = [null]"
+                <oui-button
+                    data-variant="secondary"
+                    data-size="l"
+                    data-on-click="AgentsOvhPabxCtrl.addAgentForm.numbers = [null]"
                     data-ng-if="AgentsOvhPabxCtrl.addAgentForm.numbers[0]"
-                ></button>
-                <button
+                >
+                    <span data-translate="cancel"></span>
```
Comment (marie-j) [inline packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/agents/ovhPabx/ovh-pabx.html:98]: Unscoped translation ?

File: packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/ovhPabx/sounds/sounds.html [hunk 8]
```diff
@@ -72,7 +65,7 @@
                     data-upload-errors="$ctrl.uploadErrors"
                 >
                     <i
```
Comment (marie-j) [inline packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/ovhPabx/sounds/sounds.html:67]: As icon is updated maybe we can `aria-hidden="true"`

File: packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/ovhPabx/sounds/translations/Messages_es_ES.json [hunk 9]
```diff
@@ -1,12 +1,12 @@
 {
   "telephony_alias_ovh_pabx_sounds_list_title": "Liste des sons",
```
Comment (marie-j) [inline packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/ovhPabx/sounds/translations/Messages_es_ES.json:2]: Maybe we need to resubmit translations to get translated content

File: packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/ovhPabx/tts/tts.html [hunk 10]
```diff
@@ -247,6 +249,7 @@ <h1 data-translate="telephony_alias_ovh_pabx_tts_title"></h1>
             <div
                 class="mt-2"
                 data-ng-if="$ctrl.tts.raw.length"
+                data-ng-show="$ctrl.tts.raw.length > itemsPerPage"
```
Comment (marie-j) [inline packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/ovhPabx/tts/tts.html:252]: Can't the two be grouped ?

File: packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/dialplan.less [hunk 11]
```diff
@@ -12,6 +13,10 @@
   margin-top: 1rem;
 }
 
+.dropdown-menu {
+  z-index: @zindex-modal-background + 100;
```
Comment (marie-j) [inline packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/dialplan.less:17]: I'll trust you on this one 😄

File: packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/rule/edit/edit.controller.js [hunk 12]
```diff
@@ -208,6 +210,78 @@ export default class DialplanExtensionRuleEditCtrl {
   onRuleActionChange() {
     this.parentCtrl.popoverStatus.move = false;
     this.rule.actionParam = '';
+    const step = this.parentCtrl.extension.position;
+
+    switch (this.rule.action) {
+      case 'bridge':
```
Comment (marie-j) [inline packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/rule/edit/edit.controller.js:216]: This saddens me just for `atInternet`

File: packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/rule/edit/edit.html [hunk 13]
```diff
@@ -752,6 +750,7 @@
                                     <span class="oui-radio__icon"></span>
                                     <span
                                         class="oui-radio__text"
+                                        style="max-width: 20rem"
```
Comment (marie-j) [inline packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/rule/edit/edit.html:753]: No other options ?

File: packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/ovhPabx/tts/tts.html (hunk 10 above)
Comment (frenautvh) [reply packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/ovhPabx/tts/tts.html:252]: in this code, the pagination is generating the items (slicing the array) so adding the condition to the 'ng-if' will no work (the pagination won't be rendered and no items will be displayed)

File: packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/rule/edit/edit.controller.js (hunk 12 above)
Comment (frenautvh) [reply packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/rule/edit/edit.controller.js:216]: yes the tracking name choice doesn't make it easy to implement cleanly :/

File: packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/rule/edit/edit.html (hunk 13 above)
Comment (frenautvh) [reply packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/rule/edit/edit.html:753]: adding a class ?
for "dirty" css like that i prefer inline style as it's straight forward when you read the code (and also it's difficult to find a relevent class name) but i'm not sure it's good practice, just personal preference. I can change it if necessary what do you think ?

File: packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/agents/ovhPabx/ovh-pabx.html (hunk 7 above)
Comment (frenautvh) [reply packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/agents/ovhPabx/ovh-pabx.html:98]: yes it's used everywhere in manager telecom ¯\_(ツ)_/¯

**Review by antleblanc — APPROVED** [review]

**Pull request merged**